include/NovelRT/Timing/StepTimer.h

```diff
@@ -83,11 +83,11 @@ namespace NovelRT::Timing {
     }
 
     static double TicksToSeconds(uint64_t ticks) {
```

Comment:
looks like were violating our naming convention in a couple places - not for this PR, but for the clang format review.